arangodb-net-standard/Serialization/JsonNetApiClientSerialization.cs

```diff
@@ -1,26 +1,24 @@
 ﻿using Newtonsoft.Json;
-using Newtonsoft.Json.Serialization;
-using System;
-using System.Collections.Generic;
 using System.IO;
-using System.Reflection;
 using System.Text;
 
 namespace ArangoDBNetStandard.Serialization
 {
     /// <summary>
     /// Implements a <see cref="IApiClientSerialization"/> that uses Json.NET.
     /// </summary>
-    public class JsonNetApiClientSerialization : IApiClientSerialization
+    public class JsonNetApiClientSerialization : ApiClientSerialization
     {
+        public override IApiClientSerializationOptions DefaultOptions => new ApiClientSerializationOptions(true, true);
```

Comment:
Would it not be better to set the default options in the API methods that are providing the options? We may want to use different default based on the method.
Example: currently in `PostDocumentAsync` we use `false, false`. Are we sure we want `true, true` to be the default now?

We could do the check in the API methods directly. Example:
```
var content = GetContent(document, options ?? new ApiClientSerializationOptions(false, false));
```

Reply:
I think its preferable to use the default options to true, true also in this case, what is the point really to not use camel case here? Why would you want to do that? when in the other cases we are using true, true. Also having ignore values to false by default it's problematic if you don't have the Ingore Attribute like in Newton. It's honestly very hard to understand the reason of having, false false in PostDocumentAsync .

If I do what you propose here (I speak for the custom serializer that I have), in my implementation I will have to remember to define the options every time in the call, which can lead to mistakes and its "harder". If we can use everywhere the default options and if it doesn't break anything, why we should get in this trouble?

If there is a corner case that needs false, false it should be dealt separately, by adding one more property in options or something. Corner cases should not dictate the overall behaviour. Also would lead to mistakes if we choose custom options.